projects/neoforge/build.gradle

```diff
@@ -42,214 +52,122 @@ checkJarCompatibility {
     baseJar = createCompatJar.flatMap { it.output }
 }
 
-installerProfile {
-    profile = 'NeoForge'
-}
-
-minecraft {
-    // FML looks for this mod id to find the minecraft classes
-    modIdentifier 'minecraft'
-
-    accessTransformers {
-        file rootProject.file('src/main/resources/META-INF/accesstransformer.cfg')
-    }
-}
+// TODO: re-enable AT validation
+//tasks.configureEach { tsk ->
+//    if (tsk.name == 'neoFormApplyUserAccessTransformer' && project.hasProperty('validateAccessTransformers')) {
+//        tsk.inputs.property('validation', 'error')
+//        tsk.logLevel('ERROR')
+//        tsk.doFirst {
+//            tsk.getRuntimeProgramArguments().addAll(tsk.getRuntimeProgramArguments().get())
+//            tsk.getRuntimeProgramArguments().add('--access-transformer-validation=error')
+//        }
+//    }
+//}
```

Comment:
Can you add this back? AT validation not being guaranteed for Neo means that we could break people's builds that use AT validation on the consumer side.